src/public/media.ts

```diff
@@ -2,48 +2,63 @@ import { GlobalVars } from '../internal/globalVars';
 import { SdkError, ErrorCode } from './interfaces';
 import { ensureInitialized, sendMessageRequestToParent, isAPISupportedByPlatform } from '../internal/internalAPIs';
 import { FrameContexts } from './constants';
+import { generateGUID } from '../internal/utils';
+import {
+  createFile,
+  decodeAttachment,
+  validateSelectMediaInputs,
+  validateGetMediaInputs,
+  validateViewImagesInput,
+} from '../internal/mediaUtil';
 
 /**
  * This is the SDK version when captureImage API is supported on mobile.
  */
 const captureImageMobileSupportVersion = '1.7.0';
 
+/**
+ * This is the SDK version when media APIs is supported on all three platforms ios, android and web.
+ */
+const mediaAPIVersion = '1.8.0';
```

Comment:
Can you also please verify that the version number in this PR in constants.ts is upgraded to '1.8.0-beta'?